integration/airflow/openlineage/airflow/dag.py

```diff
@@ -71,6 +71,10 @@ def __init__(self, *args, **kwargs):
             macros = kwargs["user_defined_macros"]
         macros["lineage_run_id"] = lineage_run_id
         kwargs["user_defined_macros"] = macros
+        if kwargs.__contains__("custom_extractors"):
```

Comment:
Mind if we keep the environment name and param name the same and use `openlineage_extractors`?

Reply:
I've used `lineage_custom_extractors` as lack of custom suggest complete override of default list.